### PR TITLE
add Jenkins badge to README

### PR DESCRIPTION
**Pull request by kibbles-n-bytes**
Expected output can be seen here: https://github.com/kibbles-n-bytes/service-catalog/tree/badge

@pmorie et al., Let me know if there's a preferred way you'd like to have the badges displayed. I cannot edit the content of the badge image itself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## `service-catalog`
 
-[![Build Status](https://travis-ci.org/kubernetes-incubator/service-catalog.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/service-catalog)
+Travis: [![Build Status](https://travis-ci.org/kubernetes-incubator/service-catalog.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/service-catalog)
+Jenkins: [![Build Status](https://service-catalog-jenkins.appspot.com/buildStatus/icon?job=service-catalog-master-testing)](https://service-catalog-jenkins.appspot.com/job/service-catalog-master-testing/)
```
Comment (arschles) [inline README.md]: @kibbles-n-bytes can you add a newline above this one, so that the badges show up on new lines in the rendered markdown?

Comment (kibbles-n-bytes) [inline README.md]: Done.

Comment (duglin) [inline README.md]: LOL I was going to ask for them all on one line :-)

**Comment by duglin** [comment]
LGTM either way

File: README.md (hunk 1 above)
Comment (pmorie) [inline README.md]: That looks super weird to have them stacked to me

**Comment by pmorie** [comment]
It looks seriously weird to have the badges stacked to me -- how about a mouseover that says which is which, and have them all on one line?

**Comment by vaikas-google** [comment]
LGTM

**Comment by kibbles-n-bytes** [comment]
I prefer them inline too, so I'm going to revert back to that.
@pmorie I did the tooltip approach; how does that look to you?

**Comment by pmorie** [comment]
Looks way better.  It seems like the height of hubris for any CI to make a badge that says 'build: passing' instead of 'my-thing-build: passing'.  THE HEIGHT

If people really want them stacked, I suggest we render a protective bag over them to avoid being the ugliest readme on github, like so:

**Comment by duglin** [comment]
I like this better than stacked.  Did you try just:
```
Travis [image]   Jenkins [image]
```
I'd be curious to see how that looked

**Comment by kris-nova** [comment]
I agree with @pmorie here.

I think taking advantage of horizontal real-estate especially at the top of a `README.md` file is critical. The last thing a dev wants is to have more noise to scroll through just to get to the **INSTALL** section of the docs.

+1 horizontal badges

**Comment by MHBauer** [comment]
Now that I see it, why's it broken on master.

**Pull request merged**